### ISSUE TITLE
No audio when device is in silent mode (Android 10)

### ISSUE THREAD

**Issue by marissarousseau** [SimpleMobileTools/Simple-Clock#204]
I'm experiencing issue [#100](https://github.com/SimpleMobileTools/Simple-Clock/issues/100) on my Google Pixel 3. My alarm audio stream is on max volume and the device is in silent mode (which just turns off the ring/notification stream), yet the alarm sound doesn't play. The alarm sound only plays when my device has the ringer turned on. Is there a way to ensure the app uses the alarm audio stream? I can't find a setting for it.

**Comment by tibbi** [comment]
It does use it already. Do other Clock apps work well?

**Comment by marissarousseau** [comment]
@tibbi yep I've never had a problem with the stock clock app. I haven't tried any other clock apps aside from stock and Simple Clock.

**Comment by tibbi** [comment]
well stocks apps usually work differently than the downloaded ones. Please try downloading one app.

**Comment by marissarousseau** [comment]
Just tried [this app](https://f-droid.org/app/com.better.alarm) and the alarm sound works fine.

**Comment by tibbi** [comment]
okay then, Ill take a look at it. Do you have Android 10?

**Comment by marissarousseau** [comment]
Thank you for looking into it. And yes I have Android 10.

**Comment by tibbi** [comment]
retry with 5.2.2 released soon please

**Comment by marissarousseau** [comment]
Will do. Thank you for your help with this issue. I use quick a few of these apps, just donated to help support!

**Comment by marissarousseau** [comment]
@tibbi I just received the 5.2.2 update via F-Droid. After multiple tests in silent mode (ring/vibrate off), the app doesn't play an alarm sound, even with the alarm audio stream being at 100% volume. It does work in both vibrate and ring modes, however.

**Comment by tibbi** [comment]
ok Ill take a look at it

**Comment by TLATER** [comment]
I'm also seeing this, and have been roughly since I started using lineage OS.

Any way we can help debug?

**Comment by oluc** [comment]
same here: muted phone = alarm is silent
(other alarm app plays sound)
(Android 8)

**Comment by oluc** [comment]
this app [1] has a setting to "Override sound profile", and the sound is played even on muted phone. Maybe this help?

[1] https://github.com/adrcotfas/Goodtime

**Comment by tibbi** [comment]
I dont know, this app wont be a priority anytime soon. It can have so many device specific issues that it isn't even worth working on.

**Comment by oluc** [comment]
trying to understand : 
is it really due to specific devices? or just normal default behavior for all? 

I mean that when user mutes the phone, alarms should not ring — unless the user specifically allows that alarm/app to overwrite "mute". That's what Goodtime [1] (a simple timer) is doing.

Are there devices where this app rings even though the phone is muted?

**Comment by tibbi** [comment]
there are many things done differently related to audio in general, but alarms also just arent reliable enough. Check Google Clock, how many bad ratings it has related to it being unreliable. https://play.google.com/store/apps/details?id=com.google.android.deskclock&showAllReviews=true

**Comment by oluc** [comment]
fair enough. thanks

**Comment by sasukeiscool** [comment]
This is still a problem in Android 11 (Galaxy S20) as well

**Comment by realmain** [comment]
Just want to say, I'm also having this issue on Android 13. My audio stream is on max volume, but if my phone is on vibrate or silent, it won't play alarm sound, it'll only vibrate.

**Comment by tibbi** [comment]
get into the notification settings of the app through your device settings and tweak it there. Maybe the app isnt allowed to ignore DND mode.
